.travis.yml

```diff
@@ -41,10 +40,8 @@ before_install:
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then   ./scripts/installMacDeps.sh --unattended --disable-services --use-cache;    fi
 
   # LINUX
-  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then ./scripts/installTravisDeps.sh && export CC="/usr/bin/gcc-5" && export CXX="/usr/bin/g++-5" ; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then ./scripts/travisInstallDeps.sh && export CC="/usr/bin/gcc-5" && export CXX="/usr/bin/g++-5" ; fi
```

Comment:
Why do we change the name here? we do have a kind of pattern with install[System]Deps.sh, am I right?